README.md

```diff
@@ -13,13 +13,68 @@ Use [Composer](https://getcomposer.org/) to install the testing framework.
 
 ```bash
 $ composer require --dev nimut/testing-framework
+$ composer require typo3/cms-core
+$ composer require typo3/cms
 ```
-
 Composer will add the package as a dev requirement to your composer.json and install PHPUnit and vfsStream as its
 dependencies.
 
+or just edit your compose.json file:
+
+```json
+...
+"require": {
+    "typo3/cms-core": ">=7.6.0,<7.6.99",
+    "typo3/cms": "^7.6"
+  },
+  "require-dev": {
+    "nimut/testing-framework": "^1.0"
+  },
+...
+```
+And run ```composer install```
```

Comment:
Wrong. If a composer.lock file exists an update is needed. Better use composer require as mentioned above.

Reply:
By versioning our composer.json, we want to run single command to install all dependencies. This way is usual in other technologies.

Reply:
Feel free to do so. However, I would recommend to add typo3/cms as a dev-dependency. And as I mentioned before, after edit your composer.json file you need to run composer update instead composer install.